core/domain/question_services.py

```diff
@@ -53,10 +53,10 @@ def _migrate_state_schema(versioned_question_state):
             <= feconf.CURRENT_STATES_SCHEMA_VERSION):
         raise Exception(
             'Sorry, we can only process v25-v%d state schemas at present.' %
-            feconf.CURRENT_STATE_SCHEMA_VERSION)
+            feconf.CURRENT_STATES_SCHEMA_VERSION)
```

Comment:
Thanks for catching this!

Actually -- it should probably be current_state_schema_version throughout, since the schema version refers to the state object and not the states dict. Could we change throughout to be consistent?